e2e/test/scenarios/admin-2/whitelabel.cy.spec.js

```diff
@@ -191,16 +191,13 @@ describeEE("formatting > whitelabel", () => {
           );
           cy.findByText(
             "The image you chose is corrupted. Please choose another one.",
-          ).should("be.visible");
+          )
+            .scrollIntoView()
```

Comment:
after updating the new design. When we select the `Custom` option, the setting height is increased and this causes the title to go off-screen. And the error message is also in the title, so we need to scroll this back into the view to assert it.